README.md

```diff
@@ -6,10 +6,91 @@
 
 ## Introduction
 
-Frequenz gRPC API for monitoring and control of microgrids.
+The Frequenz Microgrid API is an service for operating and monitoring electrical
+components and sensors within a microgrid. It offers various functionalities
+related to microgrid management, including metadata retrieval, component
+listing, setting power levels, and controlling the state of components. The API
+is structured around the concept of a microgrid, which is a localized energy
+system composed of various interconnected components and auxiliary sensors.
+Microgrids are typically connected to the main grid, but can also operate
+independently, in which case they are referred to as _islanded_ microgrids.
 
-This repository provides the core gRPC and protobuf spec together with
-support for generating dedicated API packages for various languages.
+The API supports different _categories_ of electrical components and sensors.
+The list of supported electrical component categories cam be found [here](https://github.com/frequenz-floss/frequenz-api-common/blob/v0.4.0/proto/frequenz/api/common/v1/components.proto#L13-L72).
+The list of supported sensor categories cam be found [here](https://github.com/frequenz-floss/frequenz-api-common/blob/83e96a9cb3fd88db06c47d29be54da435fc648e3/proto/frequenz/api/common/v1/sensors.proto#L13-L38).
+Note that each of these categories could have their own _types_, e.g., inverters
+could belong to the type "battery" or "PV". This information is stored in the
+metadata of each component and sensor, and is exposed by the API.
+
+The API streams data from the microgrid in real-time, allowing users to retrieve
+the status and performance of components and sensors. It also allows users to
+control the microgrid by setting power levels and controlling the state of
+components.
+
+## Objective
+
+The objective of this API is to provide a standardized interface for monitoring
+and controlling components and sensors within a microgrid. The API is designed
+to be vendor-agnostic, allowing users to interact with components from various
+vendors using the same API.
+
+### Key Features
+
+- **Metadata Retrieval:** The API allows users to retrieve metadata about the
+  microgrid, providing information about the microgrid's unique identifier (ID)
+  and geographical coordinates.
+
+- **Component Listing:** Users can list components in the microgrid, optionally
+  filtered by component IDs and categories.
+
+- **Component Graphs:** The API provides a way to list connections between
+  components. Connections are represented as pairs of component IDs, describing
+  an abstract flow of current according to the passive sign convention. The
```

Comment:
The lines you are referring to do not talk about graphs at all. How this turns into a graph is explained in the next line.

Reply:
I added the term `directed` in the next line where it mentions graphs.

Reply:
> Furthermore I'd suggest to write ... "an abstracted flow of current ..." because I understand that there is an "abstract flow". Also we are following the passive the passive sign convention.

Difficult to follow you here. I updated the text again. Note that we want to keep it high-level and not add too much tech detail here.